.github/workflows/ci.yaml

```diff
@@ -47,412 +91,422 @@ jobs:
+        with:
+          images: ${{ env.DOCKER_NAMESPACE }}/parser-${{ matrix.parser }}
+          tag-sha: true
+          tag-semver: |
+            {{ version }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: securecodebox/update-field
-          path: ./hooks/update-field/
-          tag_with_ref: true
-          tag_with_sha: true
-          build_args: baseImageTag=ci-local
-  scannerImages:
-    # Note we only build images for scanner that don't provider official public container images
-    name: "Build / Scanner"
+          password: ${{ secrets.DOCKER_TOKEN }}
+
+      - name: Set baseImageTag to commit hash
+        run: |
+          echo "baseImageTag=sha-$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+
+      - name: Build and Push
+        uses: docker/build-push-action@v2
+        with:
+          context: ./scanners/${{ matrix.parser }}/parser
+          file: ./scanners/${{ matrix.parser }}/parser/Dockerfile
+          build-args: |
+            namespace=${{ env.DOCKER_NAMESPACE }}
+            baseImageTag=${{ env.baseImageTag }}
+          platforms: linux/amd64
+          push: true
+          tags: ${{ steps.docker_meta.outputs.tags }}
+          labels: ${{ steps.docker_meta.outputs.labels }}
+
+  # ---- Build | Scanners ----
+
+  # Note we only build images for scanner that don't provider official public container images
+
+  # ---- Build | Scanners | Third Party Scanner ----
+
+  # This Matrix should contain Third Party Scanners
+  # The Tag for the Image should be the current version of the Scanner
+
+  scanners-third-party:
+    name: "Build | Third Party Scanner"
+    needs: [unit-python, unit-javascript]
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        scanner: [gitleaks, kube-hunter, kubeaudit, ncrack, nmap]
+
     steps:
-      - uses: actions/checkout@master
-      - uses: docker/build-push-action@v1
-        name: "Build & Push Gitleaks Scanner Image"
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: securecodebox/scanner-gitleaks
-          path: ./scanners/gitleaks/scanner/
-          tags: "v6.1.2,latest"
-      - uses: docker/build-push-action@v1
-        name: "Build & Push Git-Repo-Scanner Image"
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set ENV Var with Scanner Version
+        # Notice: The current version of the scanner is provided via the Chart.yaml to ensure
+        # there is only one place to edit the version of a scanner
+        run: |
+          cd scanners/${{ matrix.scanner }}
+          echo "scannerVersion=$(cat Chart.yaml | grep appVersion | cut -d" " -f2)" >> $GITHUB_ENV
```

Comment:
This seems to break if the appVersion string in the Chart.yaml is surrounded by `"`'s.
At least `echo "\"foobar\"" | cut -d" " -f2` seems to be still outputting `"foobar"` or am I recreating this wrong?